sql-client/Dockerfile

```
@@ -21,26 +21,26 @@
 #SQL CLI - inspired by https://github.com/wuchong/flink-sql-demo/tree/v1.11-EN/sql-client
 ###############################################################################
 
-FROM flink:1.15.0-scala_2.12-java11
+FROM flink:1.16.0-scala_2.12-java11
 
 # Create CLI lib folder
 COPY bin/* /opt/sql-client/
```

The error I got was due to copying the sql client from the wrong path, this solves it

```suggestion
COPY sql-client/bin/* /opt/sql-client/
```